modules/core/src/main/resources/reference.conf

```diff
@@ -42,6 +42,11 @@
         "hoodie.metadata.index.column.stats.column.list": "load_tstamp,collector_tstamp,derived_tstamp,dvce_created_tstamp"
         // "hoodie.embed.timeline.server.reuse.enabled": "true" // TODO: Experiment with this.
       }
+
+      "catalog": {
+        "type": "Hadoop"
+        "options" {}
```

Comment:
`"options": {}` ?

Reply:
Whoops, thanks.  Surprisingly, that's actually valid hocon syntax!  But I should change it anyway.

Reply:
I wasn't 100% sure but good to learn, thanks :D